java/core/src/main/java/co/worklytics/psoxy/gateway/impl/oauth/ClientCredentialsGrantTokenRequestBuilder.java

```diff
@@ -9,6 +9,7 @@
 import com.google.api.client.json.webtoken.JsonWebSignature;
 import com.google.api.client.json.webtoken.JsonWebToken;
 import com.google.common.annotations.VisibleForTesting;
+import io.opencensus.trace.Link;
```

Comment:
used?

Reply:
ah, no; i think came from newrelic branch where this was originally implemented.